### DISCUSSION TITLE
Where is the source code of the docs?

### DISCUSSION THREAD

**Discussion by tordans** [protomaps/PMTiles#245] [Q&A] [answered]
I want to contribute or suggest a change to https://protomaps.com/docs/frontends/maplibre#react but I cannot find the docs on Github. Are they private?  

Should I instead create an issue on this repo?

**Comment by bdon** [comment, marked as answer]
They are part of the larger repo which backs https://protomaps.com, which is not on GitHub.

I agree ideally in the long term the docs part alone should be on GitHub, probably through one of two approaches:

* Each repo in https://github.com/protomaps/ has a separate docs set up via GH pages, advantages: less docs drift, disadvantages: docs are scattered around
* We have a central `protomaps/docs` repo

Can you include your changes here for now so I can update it?

**Reply by tordans** [reply]
>Can you include your changes here for now so I can update it?

My first suggestion would be on https://protomaps.com/docs/frontends/maplibre#vector-basemaps add the URL to the first code snipped

```diff
style: {
    version:8,
    glyphs:'https://cdn.protomaps.com/fonts/pbf/{fontstack}/{range}.pbf',
    sources: {
        "protomaps": {
            type: "vector",
-            url: "...",
+            url: "pmtiles://https://example.com/example.pmtiles",
            attribution: '<a href="https://protomaps.com/">Protomaps</a> © <a href="https://openstreetmap.org/">OpenStreetMap</a>'
        }
    },
    layers: layers("protomaps","light")
}
```

My main issue was: I was having super weird console errors that I could not google to fix…
The reason was, I was missing the weird double protocol on the url `pmtiles://https…`.
I think I might have found this if there where a URL in this place…

However, this is not the only solution that would be great to clarify this.

Maybe something more like 

```diff
        "protomaps": {
            type: "vector",
-            url: "...",
+            url: "pmtiles://https://example.com/example.pmtiles",
+            //         ^-- Remember to prefix the URL with `pmtiles://`
            attribution: '<a href="https://protomaps.com/">Protomaps</a> © <a href="https://openstreetmap.org/">OpenStreetMap</a>'
        }
```

**Reply by tordans** [reply]
And yes, making the docs accessible to contributions would be great. Maybe https://tailwindui.com/templates/syntax is a start to have it in a separate repo…

**Reply by bdon** [reply]
I've been eyeing https://github.com/vuejs/docs as a good example to work off of

**Reply by bdon** [reply]
It's updated on https://protomaps.com/docs/frontends/maplibre now. 

I have created http://github.com/protomaps/docs for the new site using VitePress. https://protomaps.github.io/docs/markdown-examples.html is a preview of what it will look like - haven't migrated any content yet. But it has a working search bar at the top! This will be useful once we have documentation for all the basemap layers and tags. Thanks!

**Reply by bdon** [reply]
https://github.com/protomaps/docs is the new home, live at https://docs.protomaps.com

Am in the process of setting up redirects.
